api/tests/routes/pro/signup_pro_V2_test.py

```diff
@@ -59,6 +64,29 @@ def test_when_user_data_is_valid(self, client):
         assert mails_testing.outbox[0]["To"] == user.email
         assert mails_testing.outbox[0]["template"] == asdict(TransactionalEmail.EMAIL_VALIDATION_TO_PRO.value)
 
+    @override_features(WIP_ENABLE_NEW_NAV_AB_TEST=True)
+    def test_user_pro_new_nav_bar_ab_(self, client):
+        data = BASE_DATA_PRO.copy()
+        emails = []
+        for _ in range(2):
+            data["email"] = "".join(random.choices(string.ascii_letters, k=12)).lower() + "@test.com"
+            client.post("/v2/users/signup/pro", json=data)
+            emails.append(data["email"])
+        users = (
+            User.query.options(joinedload(User.pro_new_nav_state))
+            .filter(User.email.in_(emails), User.pro_new_nav_state.has())
+            .all()
+        )
+        assert len(users) == 1
+        # for invited user
+        invited_email = "titi@test.com"
+        user_inviter = users[0]
+        OffererInvitationFactory(user=user_inviter, email=invited_email)
+        data["email"] = invited_email
+        client.post("/v2/users/signup/pro", json=data)
+        pro_new_nav_state = User.query.filter_by(email=invited_email).first().pro_new_nav_state
```

Comment:
[détail] Il n'y a qu'un seul utilisateur avec cet e-mail. `first()` fonctionne aussi, évidemment, mais le lecteur pourrait se demander pourquoi on n'utilise pas `one()` : 

```suggestion
        pro_new_nav_state = User.query.filter_by(email=invited_email).one().pro_new_nav_state
```